modules/workflows/OrcaCopyToGlacierWorkflow/orca_copy_to_glacier_workflow.asl.json

```diff
@@ -7,7 +7,7 @@
         "cma": {
           "event.$": "$",
           "task_config": {
-            "excludeFileTypes": "{$.meta.collection.meta.excludeFileTypes}",
+            "OrcaExcludedFileRegEx": "{$.meta.collection.orca.OrcaExcludedFileRegEx}",

```

Comment:
We should probably keep casing rules. "orcaExcluded..."

Reply:
agreed